modules/standard/IO.chpl

```diff
@@ -2630,7 +2631,7 @@ proc _channel.advance(amount:int(64)) throws {
    Reads until ``byte`` is found and then leave the channel offset
    just after it.
 
-   :throws EofError: if the requested `byte` could not be found.
+   :throws UnexpectedEofError: if the requested `byte` could not be found.
```

Comment:
Out of curiosity, was the behavior changed here, or was the old documentation incorrect?